examples/ping-ring/main.go

```diff
@@ -95,12 +95,19 @@ func main() {
 
 	logger := log.StandardLogger()
 
+	rp, err := ringpop.New("ping-app",
+		ringpop.Channel(ch),
```

Comment:
Just leaving a note here to say: after some discussion and prototyping, this is proving too difficult to do right now. We looked at moving `Option`s to a `config` package but this resulted in circular dependencies that would take significant time effort to resolve.